Program.cs

```diff
@@ -41,6 +44,7 @@ public static void PersonList()
             }
```

Comment:
You can use string interpolation instead of these too many pulses!

Reply:
done 👍